### DISCUSSION TITLE
Launch with saved config? bat or profile saving?

### DISCUSSION THREAD

**Discussion by DisasterofPuppets** [lllyasviel/Fooocus#1592] [General]
HI all,

I've only just started using AI generated images and this is great. 

I was wondering if there is a way to launch Fooocus with specific settings (or save a workspace setting)?

For example, When I launch (python.exe -s Fooocus\entry_with_update.py --preset realistic) 
I generally switch the performance to Quality, and choose additional lora packs.

Is there a way to launch or save my workspace without having to set these manually every time?

Thank you.

**Comment by xhoxye** [comment]
Look at the files in the Fooocus\presets folder and you'll find something.

**Reply by DisasterofPuppets** [reply]
champ thank you

**Comment by Amit30swgoh** [comment]
> HI all,
> 
> I've only just started using AI generated images and this is great.
> 
> I was wondering if there is a way to launch Fooocus with specific settings (or save a workspace setting)?
> 
> For example, When I launch (python.exe -s Fooocus\entry_with_update.py --preset realistic) I generally switch the performance to Quality, and choose additional lora packs.
> 
> Is there a way to launch or save my workspace without having to set these manually every time?
> 
> Thank you.

Same question pls how can we save it via lora ?

**Reply by DisasterofPuppets** [reply]
Navigate to your install directory and open the Presets folder.
Copy one of the presets and rename to your ownpreset.json
Edit with notepad etc and change the settings as required.

For example, if you want Loras to be selected by default, you can simply replace or add them to the lines below

```
 "default_model": "realisticStockPhoto_v10.safetensors",
    "default_refiner": "",
    "default_refiner_switch": 0.5,
    "default_loras": [
        [
            "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors",
            0.25
        ],
        [
            "YOUR LORA",
            1.0
        ],
```

Once saved to launch you just need to copy the run_realistic.bat files in the main install directory, rename it and edit it so the switch at the end matches your newly created preset

eg

.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset ownpreset
pause
